src/pages/data-visualization/chart-types/index.mdx

```diff
@@ -16,8 +16,8 @@ Start by identifying the purpose of the visualization and choose a chart type ap
 <Row className="overview-card-group">
 <Column  colMax={3} colXl={3} colLg={4} colMd={4} colSm={2} noGutterSm >
  <OverviewCard
-    title="Bar chart"
-    href="/data-visualization/basic-charts#bar-chart"
+    title="Simple bar (vertical)"
+    href="/data-visualization/basic-charts#simple-bar-(discrete)"
```

Comment:
most of the OverviewCards link to `h2`s but others link to the chart demo wrappers directly. should they all follow the same pattern (or a similar pattern, linking to a description first rather than the chart demo itself)? (cc @janchild for content review)